File: app/src/main/kotlin/br/com/sailboat/medzy/view/adapter/recycler_item/SubheaderRecyclerItem.java
package br.com.sailboat.medzy.view.adapter.recycler_item;

import br.com.sailboat.canoe.recycler.RecyclerItem;
import br.com.sailboat.medzy.view.adapter.ViewType;

public class SubheaderRecyclerItem implements RecyclerItem {

    @Override
    public int getViewType() {
        return ViewType.SUBHEADER;
    }

}

File: app/src/main/kotlin/br/com/sailboat/medzy/view/adapter/recycler_item/SubheaderRecyclerItem.kt
package br.com.sailboat.medzy.view.adapter.recycler_item

import br.com.sailboat.canoe.recycler.RecyclerItem
import br.com.sailboat.medzy.view.adapter.ViewType

data class SubheaderRecyclerItem(var subheaderText: Int) : RecyclerItem {

    override fun getViewType(): Int {
        return ViewType.SUBHEADER
    }

}
